### PR TITLE
Added resources section to `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by Snailedlt**
Added a new section to `CONTRIBUTING.md` like we discussed in the Discord channel.
@Panquesito7 Take a look if you want :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ First of all, thanks for taking the time to contribute! This project can only gr
   <li><a href="#bugs">Common Bugs and Solutions</a></li>
   <li><a href="#discordServer">Discord server</a></li>
   <li><a href="#release">Release strategy, conventions, preparation and execution</a></li>
+  <li><a href="#resources">Resources and tools we use and recommend</a></li>
```
Comment (Thomas-Boi) [inline CONTRIBUTING.md:23]:
```suggestion
  <li><a href="#resources">Recommended resources and tools</a></li>
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -490,3 +491,13 @@ We are running a Discord server. You can go here to talk, discuss, and more with
     <li>Create a <a href="https://github.com/devicons/devicon/releases/new">new release</a> using the format "<b>Release v<i>MAJOR</i>.<i>MINOR</i>.<i>PATCH</i></b>" as tag and release title. Use the earlier created description as description of the release.</li>
     <li>Publishing the release will trigger the <a href="/.github/workflows/npm_publish.yml">npm_publish.yml</a> workflow which will execute a <code>npm publish</code> leading to a updated <a href="https://www.npmjs.com/package/devicon">npm package</a> (v<i>MAJOR</i>.<i>MINOR</i>.<i>PATCH</i>).</li>
 </ol>
+
+<h2 id='resources'>Resources and tools we use and recommend</h2>
```
Comment (Thomas-Boi) [inline CONTRIBUTING.md:495]:
```suggestion
<h2 id='resources'>Recommended resources and tools</h2>
```

**Review by Thomas-Boi — COMMENTED** [review]
The changes look good to me. Clean and concise design. The only change I'd suggest is shortening the title since the old one is a bit clunky.

**Comment by Snailedlt** [comment]
> The changes look good to me. Clean and concise design. The only change I'd suggest is shortening the title since the old one is a bit clunky.

Agreed and done! :D

**Review by Thomas-Boi — APPROVED** [review]
Everything looks great to me 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -490,3 +491,13 @@ We are running a Discord server. You can go here to talk, discuss, and more with
     <li>Create a <a href="https://github.com/devicons/devicon/releases/new">new release</a> using the format "<b>Release v<i>MAJOR</i>.<i>MINOR</i>.<i>PATCH</i></b>" as tag and release title. Use the earlier created description as description of the release.</li>
     <li>Publishing the release will trigger the <a href="/.github/workflows/npm_publish.yml">npm_publish.yml</a> workflow which will execute a <code>npm publish</code> leading to a updated <a href="https://www.npmjs.com/package/devicon">npm package</a> (v<i>MAJOR</i>.<i>MINOR</i>.<i>PATCH</i>).</li>
 </ol>
+
+<h2 id='resources'>Recommended resources and tools</h2>
+
+| Tool Name          | Link                                               | Description & Usage                                |
+| :----------------- | :------------------------------------------------- | :------------------------------------------------- |
+| Inkscape           | https://inkscape.org/                              | Desktop application for editing and Making SVG's   |
```
Comment (Panquesito7) [inline CONTRIBUTING.md:499]:
```suggestion
| Inkscape           | https://inkscape.org/                              | Desktop application for editing and making SVG's   |
```

Comment (Snailedlt) [reply CONTRIBUTING.md:499]: already merged :S

Comment (Panquesito7) [reply CONTRIBUTING.md:499]: Want to make another PR to fix this? :)

Comment (Snailedlt) [reply CONTRIBUTING.md:499]: Yes, but I don't think I have time until next year. Feel free to make a PR if you want the change before that :)

Comment (Thomas-Boi) [reply CONTRIBUTING.md:499]: It's alright. I'm updating the docs and converting it into the GitHub wikis anyway. I'll watch out for this.

**Pull request merged**